docs/lib/debugging/fragments/android/dev-menu/setup.md

```diff
@@ -0,0 +1,12 @@
+Run the following commands to add a copy of the file `amplify/.config/local-env-info.json` to the 
+directory `app/src/main/res/raw` in your Android project and rename the copy to `localenvinfo.json`:
```

Comment:
More of a product question: isn't it weird that on one hand we use `-` to separate a file name while in another place we concatenate them?

Reply:
`local-env-info.json` is an existing file and the copy added to the `app/src/main/res/raw` directory cannot be named `local-env-info.json` because Android does not allow file names with a '-' in that directory.